topics/backbonejs/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: backbone, backbonejs
+created_by: Jeremy Ashkenas
+display_name: BACKBONE.JS
+logo: backbonejs.png
+released: April 5, 2016
```

Comment:
I think we should go with the initial release date here. Even though this project hasn't been updated in a while, this release date could be obsolete with the next release.

```suggestion
released: October 13, 2010
```